src/splitInteger.test.js

```diff
@@ -8,12 +8,26 @@ describe(`Function 'splitInteger':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(splitInteger()).toBeInstanceOf(Array);
   });
 
   it(`should return an array with 'num' if parts = 1`, () => {
+    expect(splitInteger(8, 1)).toEqual([8]);
+  });
+
+  it(`should return an empty array when num = 8, parts = 'string'`, () => {
+    expect(splitInteger(8, 'string')).toEqual([]);
+  });
+
+  it(`should return an empty array, num = 'string', parts = 'string'`, () => {
+    expect(splitInteger('string', 'string')).toEqual([]);
+  });
```

Comment:
The description of the function says:
`the input to your function will always be valid`

`'string'` is an invalid input, so this test is not needed.